ubuntu/check-formatting.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+FILES=$(find src -regex ".*\.\(hpp\|cpp\|cc\|c\|h\)$")
```

Comment:
It looks like this script can also be run on MacOS without problems. Probably it is better to store such scripts to some common folder (for example we can name it 'unix').

Reply:
Sure.